tests/test_jobs.py

```diff
@@ -502,7 +516,9 @@ def test_backfill_depends_on_past(self):
         """
         Test that backfill respects ignore_depends_on_past
         """
+        clear_runs()
```

Comment:
all these clear_runs call seem very redundant. Could we move in setup phase?

Reply:
This was a bit the first attempt of fixing all the failing tests. Did now clean all those calls inside BackfillJobTest, see also: 
https://github.com/apache/airflow/pull/4396/commits/00ddeeca3030231441be604939eb80b19dcca34a